addons/crm/i18n/crm.pot

```diff
@@ -3674,22 +3674,22 @@ msgstr ""
 
 #. module: crm
 #: model_terms:ir.ui.view,arch_db:crm.crm_action_helper
-msgid "Use the top left <i>Create</i> button, or send an email to"
+msgid "Use the <i>New</i> button, or send an email to"
```

Comment:
Normally in stable we keep the old translation and add the new one (to be cleaned when reaching master and generating final pot files before release). Indeed people could fetch transifex translations without updating the code and they would still need the old translations. Could you add the new one while keeping the old one please ?

Reply:
Hello @tde-banana-odoo 

Suggested changes have been done.

Thanks!!

Reply:
Did you generate the file using the tool ? Seems the ordering is not correct, not sure ...

Reply:
Hello @tde-banana-odoo 

You can have a look now.

Thanks!!